varnish-sys/build.rs

```diff
@@ -14,10 +14,17 @@ fn main() {
             {
                 Ok(l) => l.include_paths,
                 Err(e) => {
-                    // FIXME: we should only do this with `docsrs` feature - it is pointless otherwise
-                    println!("no system libvarnish found, using the pre-generated bindings {e}");
-                    std::fs::copy("src/bindings.rs.saved", out_path).unwrap();
-                    return;
+                    // See https://docs.rs/about/builds#detecting-docsrs
+                    if env::var("DOCS_RS").is_ok() {
+                        eprintln!("libvarnish not found, for docs using the saved bindings {e}");
```

Comment:
phrasing is a bit weird, isn't it?
apart from that, all good

Reply:
oops, fixed

Reply:
honestly, it could have been me, language doesn't language good sometimes :-D